modules/spo-container-profile-instances.adoc

```diff
@@ -43,6 +46,11 @@ spec:
       app: sp-record
 ----
 
+. Label the namespace
```

Comment:
I've noticed that in https://file.rdu.redhat.com/jbrigman/namespace/security/security_profiles_operator/spo-selinux.html#spo-container-profile-instances_spo-selinux the run in to the code block varies, e.g.

```
- Label the namespace
- Create the workload:
- To record the individual profiles, delete the deployment by running the following command:
- To view the permissions used by any of the containers, run the following command:
```

It might be out of scope for this PR except for the command you've added, but it would be nice if these were consistently formatted, e.g.

```
- Label the namespace by running the following command:
- Create the workload by running the following command:
- To record the individual profiles, delete the deployment by running the following command:
- View the permissions used by any of the containers by running the following command:
```

Reply:
This is done, thank you!